src/utils/room/setPhase.ts

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import RightPanelStore from "../../stores/right-panel/RightPanelStore";
+import { IRightPanelCardState } from "../../stores/right-panel/RightPanelStoreIPanelState";
+import { RightPanelPhases } from "../../stores/right-panel/RightPanelStorePhases";
+
+/**
+ * Helper to toggle a right panel view.
```

Comment:
"toggle" means "turn on and off"? So if we call it twice with the same `phase` it turns it on once and then off again?

Reply:
Yes

Reply:
ok. Would you mind writing that down in the documentation? 

Also: what happens if we call it twice with different phases? Is the first phase left open but "hidden" by the new phase?

What happens if we call it twice with the same phase but different `cardState`s?